buildchain/buildchain/constants.py

```diff
@@ -17,16 +17,16 @@
 CMD_WIDTH: int = 14
 
 # URLs of the main container repositories.
+K8S_REPOSITORY: str = "registry.k8s.io"
 BITNAMI_REPOSITORY: str = "docker.io/bitnami"
 CALICO_REPOSITORY: str = "docker.io/calico"
-COREDNS_REPOSITORY: str = "k8s.gcr.io/coredns"
+COREDNS_REPOSITORY: str = f"{ K8S_REPOSITORY }/coredns"
```

Comment:
Did Black run on this file? Or did the rules change recently? IIRC, we would not have had spaces inside the curly braces, but I may be wrong.

Reply:
Yes black has ran on it, but apparently he don't care space or no space ....

But indeed normally there is no space, I will change it